optaplanner-docs/src/main/asciidoc/Chapter-Quick_start/Chapter-Quick_start.adoc

```diff
@@ -71,160 +69,27 @@ image::Chapter-Quick_start/cloudOptimizationValueProposition.png[align="center"]
 [[cloudBalancingDomainModelDesign]]
 === Domain Model Design
 
-Beginning with the domain model:
 
+Using a <<domainModelingGuide,domain model>> helps determine which classes are planning entities and which of their properties are planning variables. Using a domain model helps to simplify contraints, improve performance, and increase flexibility for future needs. 
```

Comment:
"Using a domain model helps ..." => There is always a domain model, even if it's implicitly, it's not optional. 
change it to "Using a good domain model helps ..."

"Using a domain model helps ..." is stated twice.